services/src/main/java/org/keycloak/protocol/oidc/endpoints/request/AuthorizationEndpointRequestParserProcessor.java

```diff
@@ -96,6 +99,18 @@ public static AuthorizationEndpointRequest parseRequest(EventBuilder event, Keyc
                 }
             }
 
+            if(Profile.isFeatureEnabled(Profile.Feature.DYNAMIC_SCOPES)) {
```

Comment:
Minor: `if` without space after. AFAIK the Keycloak code convention is something like `if (Profile.isFeatureEnabled...` . This applies to multiple places of this PR

Reply:
Apparently I didn't run the formatter on existing files to avoid introducing too many changes. Interestinly, even with the Eclipse Formatter using Wildfly's profile, the changes to most files are quite big.

Anyway, fixed those formatting issues.